wagtail/tests/testapp/blocks.py

```diff
@@ -30,3 +30,13 @@ class SectionBlock(blocks.StructBlock):
     class Meta:
         icon = "form"
         template = 'tests/blocks/section_block.html'
+
+
+class PostsStaticBlock(blocks.StaticBlock):
+    def __init__(self, *args, **kwargs):
+        super(PostsStaticBlock, self).__init__(*args, **kwargs)
+        self.admin_html = "<b>{}</b> - This block doesn't need to be configured, it will be displayed automatically".format(self.meta.label)
```

Comment:
`self.label` should be used over `self.meta.label`. `self.label` uses `self.meta.label` if it is set, otherwise it generates a label from the name of the block.